bigbone/src/main/kotlin/social/bigbone/api/Scope.kt

```diff
@@ -206,4 +245,105 @@ class Scope(private vararg val scopes: Name) {
     }
 
     override fun toString(): String = scopes.distinct().joinToString(separator = " ", transform = { it.name() })
```

Comment:
Now that we use the scope delimiter `" "` in multiple places, it might make sense to make this a local constant `private const val SCOPE_DELIMITED: String = " "` and use it in the three places we use it within this file.

Reply:
Will do, thanks for all enhancements. :)